plugins/factory-plugin/src/file-uri.ts

```diff
@@ -30,3 +31,7 @@ export default function convertToFileURI(file: string, rootFolder?: string): str
     return `file://${rootFolder}${file}`;
```

Comment:
`rootFolder` is optional but there is no check on `undefined`.

Reply:
ok will add a test :) thanks

Reply:
For this part, see line 16 vitally and there is a test as well ...

Reply:
Yep, indeed. Git collapsed part of code and it confused me.